db/migrate/20161130153206_add_last_updated_at_to_content_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastUpdatedAtToContentItems < ActiveRecord::Migration[5.0]
+  def change
+    add_column :content_items, :last_updated_at, :datetime
```

Comment:
Yes, that was part of the story. But as part of the story, if we find out that the field name is `public_updated_at` it might be good to mimic in our models. In general, the code will be more readable.